src/components/Section/Foreign/Travel/TravelDays.jsx

```diff
@@ -1,93 +1,90 @@
 import React from 'react'
 import { i18n } from '../../../../config'
-import { ValidationElement, CheckboxGroup, Checkbox } from '../../../Form'
+import { ValidationElement, RadioGroup, Radio } from '../../../Form'
 
 export default class TravelDays extends ValidationElement {
   constructor(props) {
     super(props)
     this.update = this.update.bind(this)
+    this.updateTravelDays = this.updateTravelDays.bind(this)
   }
 
-  update(response) {
-    if (!this.props.onUpdate) {
-      return
-    }
-
-    let selected = response.value
-    let list = [...(this.props.values || [])]
-
-    if (list.includes(selected)) {
-      list.splice(list.indexOf(selected), 1)
-    } else {
-      list.push(selected)
+  update(updateValues) {
+    if (this.props.onUpdate) {
+      this.props.onUpdate({
+        value: this.props.value,
+        ...updateValues
+      })
     }
+  }
 
-    this.props.onUpdate({ values: list })
+  updateTravelDays(values) {
```

Comment:
Seeing this, I'm not really sure we need both update functions. we can probably get away with something like this:

```
update(values) {
  const { onUpdate } = this.props;
  onUpdate && onUpdate({ value: values.value })
}
```